doc_template/examples_root/examples/nb/visual_behavior_ophys_dataset_manifest.ipynb

```diff
@@ -59,172 +59,81 @@
      "name": "stdout",
```

Comment:
<p>Setting a low alpha to the points on the scatter plot can show the density of the points.</p><p><a href="https://stackoverflow.com/questions/19064772/visualization-of-scatter-plots-with-overlapping-points-in-matplotlib" target="_blank">python - Visualization of scatter plots with overlapping points in matplotlib - Stack Overflow</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/AllenSDK/pull/2620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='173'/>

Reply:
Thanks, I added that